sample/src/main/java/io/ona/kujaku/sample/activities/WmtsActivity.java

```diff
@@ -0,0 +1,101 @@
+package io.ona.kujaku.sample.activities;
+
+import android.os.Bundle;
+import android.util.Log;
+
+import com.mapbox.mapboxsdk.Mapbox;
+
+import io.ona.kujaku.exceptions.WmtsCapabilitiesException;
+import io.ona.kujaku.listeners.WmtsCapabilitiesListener;
+import io.ona.kujaku.sample.BuildConfig;
+import io.ona.kujaku.sample.R;
+import io.ona.kujaku.services.WmtsCapabilitiesService;
+import io.ona.kujaku.wmts.model.WmtsCapabilities;
+import io.ona.kujaku.views.KujakuMapView;
+
+public class WmtsActivity extends BaseNavigationDrawerActivity {
+
+    private static final String TAG = WmtsActivity.class.getName();
+
+    private KujakuMapView kujakuMapView;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        Mapbox.getInstance(this, BuildConfig.MAPBOX_SDK_ACCESS_TOKEN);
+
+        kujakuMapView = findViewById(R.id.wmts_mapView);
+
+        WmtsCapabilitiesService wmtsService = new WmtsCapabilitiesService(getString(R.string.wmts_capabilities_url));
```

Comment:
The code does not seem to work with this WMTS style https://openlayers.org/en/latest/examples/data/WMTSCapabilities.xml which has the `isDefault` on the default style. MapboxGL gets a `400` error and I cannot see the expected features on the map. The expected rendered map is here https://openlayers.org/en/latest/examples/wmts-layer-from-capabilities.html

Reply:
Ok i found the issue. It comes from the {style} tag in the templare URL. All the WMTS capabilities I have seen contain tags with first upper case letter in the template URL, {Style} like the others : https://koordinates-tiles-a.global.ssl.fastly.net/services;key=d740ea02e0c44cafb70dce31a774ca10/tiles/v4/layer=7328,**{style}**/{TileMatrixSet}/{TileMatrix}/{TileCol}/{TileRow}.png

I will fix the issue to not take care about the upper or lower case tags. Thanks to have found this issue.

Reply:
I need to change the way i create the final template Url to avoid this behaviour

Reply:
From the specification, as a template processor program
> The template processor SHALL substitute the variable names by the values shown in Table 34

The style variable has been defined as both `style` and `Style`. The rest look the same. You can take a look and tell me if I left something out

Reply:
you're right. i'm wondering if it is not a typo in the specification ?? So, i suggest to accept both writing for "style" and "Style" and don't modifiy the rest. Is it ok for you ?

Reply:
Yea, this is OK